.github/workflows/renovate-approve-merge.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow is provided via the organization template repository
+#
+# https://github.com/nextcloud/.github
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
+
+name: Auto approve renovate PRs
+
+on:
+  pull_request_target:
+    branches:
+      - main
+      - master
+      - stable*
+
+permissions:
+  contents: read
+
+concurrency:
+  group: renovate-approve-merge-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  auto-approve-merge:
+    if: github.actor == 'renovate[bot]'
+    runs-on: ubuntu-latest
+    permissions:
+      # for hmarr/auto-approve-action to approve PRs
+      pull-requests: write
+      # for alexwilson/enable-github-automerge-action to approve PRs
+      contents: write
+
+    steps:
+      - uses: mdecoleman/pr-branch-name@bab4c71506bcd299fb350af63bb8e53f2940a599 # v2.0.0
+        id: branchname
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      # GitHub actions bot approve
+      - uses: hmarr/auto-approve-action@v3
+        if: startsWith(steps.branchname.outputs.branch, 'renovate/')
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      # Enable GitHub auto merge
+      - name: Auto merge
```

Comment:
Difference to the other action (for dependabot) is that it has a limit on only minor version updates https://github.com/nextcloud/text/blob/8a8d84c5c9b27e20aa7e2a97b6d09517019b1563/.github/workflows/dependabot-approve-merge.yml#L39

I would feel ok to have this for all in text due to our test coverage but not sure about making this a general action template then with that.

Reply:
I think major version bumps are really rare. But I might be wrong. How about we start with limited to minor and then remove the restriction if we they become an issue?

Reply:
I think the new actions don't know about the renovate version numbers so we cannot limit unless we find (or write an action) to check that, but fine as it is for now I'd say.

Reply:
Ah.. now i got it. So now we don't have the limit. Fine with me aswell.